fulp_modules/Z_edits/container_init/closet_init.dm

```diff
@@ -0,0 +1,44 @@
+/obj/structure/closet/secure_closet/engineering_chief/Initialize()
+	new /obj/item/clothing/shoes/workboots/digitigrade(src)
+	. = ..()
+
+/obj/structure/closet/secure_closet/research_director/Initialize()
+	new /obj/item/clothing/shoes/laceup/digitigrade(src)
+	new /obj/item/card/id/departmental_budget/sci(src) //Used in science_budget.dm
+	. = ..()
+
+/obj/structure/closet/secure_closet/chief_medical/Initialize()
+	new /obj/item/clothing/shoes/brown/digitigrade(src)
+	. = ..()
+
+/obj/structure/closet/secure_closet/hop/Initialize()
+	new /obj/item/clothing/shoes/brown/digitigrade(src)
+	. = ..()
+
+/obj/structure/closet/secure_closet/hos/Initialize()
+	new /obj/item/clothing/shoes/jackboots/digitigrade(src)
+	. = ..()
+
+/obj/structure/closet/secure_closet/captains/Initialize()
+	new /obj/item/clothing/shoes/brown/digitigrade(src)
+	. = ..()
+
+/// Security locker ID edit - Changes it to ACCESS_WEAPONS so Deputies no longer have access to it
+/obj/structure/closet/secure_closet/security
+	req_access = list(ACCESS_WEAPONS)
```

Comment:
Since this is an assignment override and not an additional variable,, it takes over the security access and allows bartenders to open sec lockers

Reply:
If you added the access as part of Initialize it would function correctly, while also extending this change to departmental security lockers

Reply:
yeah ok problem?

Reply:
I'm fine with bartenders being able to open sec lockers since they don't have access to any of the rooms that contain sec lockers, at least to my knowledge. This is better than deputies having access to lockers they shouldn't.

Reply:
My suggestion works get trolled so hard 
![image](https://user-images.githubusercontent.com/83368538/127028378-8d22a9c7-3833-4b42-93d4-7a4b1aaf76d7.png)
![image](https://user-images.githubusercontent.com/83368538/127028426-4515269c-ab56-4f29-af3d-d22f86a90209.png)